core/include/cubos/core/ecs/cubos.hpp

```diff
@@ -327,6 +343,12 @@ namespace cubos::core::ecs
 
         /// @brief Maps registered tags to their data.
         std::unordered_map<std::string, TagInfo> mTags;
+
+        /// @brief Port to listen on, in case debugging is enabled.
+        memory::Opt<uint16_t> mDebugPort;
+
+        /// @brief Type server used in case debugging is enabled.
+        reflection::TypeServer mTypeServer;
```

Comment:
I'm not sure about this but since the mDebugPort is an opt because is only used when debugging is enabled shouldn't mTypeServer also be an opt?

Reply:
Good point! Will change it to an Opt :relieved: